core/domain/exp_services.py

```diff
@@ -1277,31 +1666,38 @@ def update_exploration_status_in_search(exp_id):
 
 
 def delete_documents_from_search_index(exploration_ids):
+    """Deletes the documents from search index. 
```

Comment:
Deletes the documents corresponding to these exploration_ids from the search index.